### PR TITLE
ZenDesk New Source - New Ticket Added To View

### LINKED ISSUE #6156: [TRIGGER] Zendesk - emit new event when ticket is added to view
**Describe the event source. What app is this for, and what event does the trigger correspond to?**
The current Zendesk app has events for ticket creation, solve, etc. Often Zendesk users will want to know when tickets reach certain teams or views. Often these are related to high-priority customers, etc. Tickets are placed in views either manually by agents, or automagically by Zendesk's own automations/triggers that the customer has defined. We would like to see an event that triggers whenever a ticket is assigned to a specific view or views.

**Please provide a link to the relevant API docs for the specific service / operation this trigger is tied to.**
https://developer.zendesk.com/api-reference/ticketing/business-rules/views/

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #6156

### CODE REVIEW

File: components/zendesk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/zendesk",
-  "version": "0.5.0",
+  "version": "0.5.1",
```
Comment (GTFalcao) [inline components/zendesk/package.json:3]:
```suggestion
  "version": "0.6.0",
```
New `minor` version for a new component being added

**Review by GTFalcao — COMMENTED** [review]
Hi @michelle0927 , thanks for your contribution, LGTM!

Left a suggestion on the version number to follow the [semantic versioning guidelines](https://pipedream.com/docs/components/guidelines/#versioning), but moving the issue forward

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Zendesk_6156.pdf](https://github.com/PipedreamHQ/pipedream/files/11471293/Zendesk_6156.pdf)

**Review by luancazarine — APPROVED** [review]

**Pull request merged**